java/clients/src/test/resources/reports/ingest/job/json/jobWithMultipleRuns.json

```diff
@@ -11,15 +11,17 @@
           "startedStatus": {
             "inputFileCount": 1,
             "startTime": "2022-10-12T10:02:00.001Z",
-            "updateTime": "2022-10-12T10:02:00.123Z"
+            "updateTime": "2022-10-12T10:02:00.123Z",
+            "isStartOfRun": true
           }
```

Comment:
We probably want to avoid including isStartOfRun in the reporting JSON. At some point we'll want to include all the status updates for the run in the JSON including the validation, but that could be done in https://github.com/gchq/sleeper/issues/794.